CONTRIBUTING.rst

```diff
@@ -16,31 +16,29 @@ on metatomic:
 
 - **git**: the software we use for version control of the source code. See
   https://git-scm.com/downloads for installation instructions.
-- **Python**: you can install ``Python`` and ``pip`` from your operating system.
+- **Python**: you can install ``Python`` and ``pip`` on your operating system.
   We require a Python version of at least 3.9.
-- **tox**: a Python test runner, cf https://tox.readthedocs.io/en/latest/. You
+- **tox**: a Python test runner, see https://tox.readthedocs.io/en/latest/. You
   can install tox with ``pip install tox``.
 
 Additionally, you will need to install the following software, but you should
 not have to interact with them directly:
 
-- **cmake**: we need a cmake version of at least 3.10.
-- **a C++ compiler** we need a compiler supporting C++11. GCC >= 5, clang >= 3.7
-  and MSVC >= 15 should all work, although MSVC is not yet tested continuously.
-
-.. _tox: https://tox.readthedocs.io/en/latest
+- **cmake**: we need a cmake version of at least 3.16.
+- **a C++ compiler** we need a compiler supporting C++11. GCC >= 7, clang >= 5
```

Comment:
Rather than having to remember to sync this with the `installation` document, maybe it is better to just delegate to that as the source of installation instructions..

Reply:
This changes very infrequently, so I think we are fine =)